py3status/core.py

```diff
@@ -1069,8 +1079,12 @@ def run(self):
                         # store the output as json
                         output[index] = out
 
-                # build output string
-                out = ",".join(x for x in output if x)
-                # dump the line to stdout
-                write(f",[{out}]\n")
+                # build output string and dump to stdout
+                out = ""
+                if self.config["py3_config"]["general"].get("output_format") == 'tmux':
+                    out = "#[fg=brightblack]|#[default]".join(x for x in output if x)
```

Comment:
(Config) I counted this in three different places. Maybe this could be evaluated just once in `setup` function.

Why `brightblack`? I wonder if we need to make this a config too because of different terminals and/or preferences.

Reply:
I refactored the config query into `setup` in 55a13da

brightblack seemed like a reasonable default at the time of writing, because it mimics the separator in i3wm. But you are right, that should probably be configurable. maybe through a tmux_separator option in the `general` section of the config file?

Reply:
I've tried to make output_format_separator configurable by passing it through the standard Formatter, but I'm having a few issues. Here's a quick illustration of what I'm trying to achieve:

in py3status.conf:
```
general {
  colors = true
  output_format = tmux
  color_separator = red # defaults to #333333
}

py3status {
  output_format_separator = "\?color=color_separator \|"
}
```

I'm currently trying to parse the output_format_separator string in `setup` like this:

```
formatted = Formatter().format(output_separator, param_dict={'color_separator': self.config["py3_config"]["general"]["color_separator"]})
```

but it's not working. I think I'm still missing a detail about how these color names are interpreted, and how they are satisfied at parse time. I'll dig a little deeper and see if I can find something. But would that general idea be okay?